README.md

```diff
@@ -82,6 +82,8 @@ sonobuoy delete --wait
 
 > Note: The --wait option ensures the Kubernetes namespace is deleted, avoiding conflicts if another Sonobuoy run is started quickly.
 
+If you have an issue with permissions in your cluster but you still want to run Sonobuoy, you can use `--aggregator-permissions` flag in `sonobuoy gen` command. Read more details about it [here][aggregator-permissions].
```

Comment:
Remove the "in `sonobuoy gen` command". The flag is for `sonobuoy run` and `sonobuoy gen`.